Adyen.Test/TerminalApiInputTest.cs

```diff
@@ -17,9 +17,9 @@ public void InputRequestTest()
                 var paymentRequest = MockPosApiRequest.CreatePosPaymentRequest();
                 //create a mock client
                 var client = CreateMockTestClientPosLocalApiRequest("mocks/terminalapi/input-request-response.json");
-                var posPaymentLocalApi = new PosPaymentCloudApi(client);
-                var configEndpoint = posPaymentLocalApi.Client.Config.Endpoint;
-                var saleToPoiResponse = posPaymentLocalApi.TerminalApiCloudSync(paymentRequest);
+                var terminalCloudApi = new TerminalCloudApi(client);
+                var configEndpoint = terminalCloudApi.Client.Config.Endpoint;
+                var saleToPoiResponse = terminalCloudApi.TerminalRequestSync(paymentRequest);
```

Comment:
should this be TerminalLocalApi because of `CreateMockTestClientPosLocalApiRequest`?